pkg/apis/componentconfig/v1alpha1/defaults.go

```diff
@@ -368,9 +368,12 @@ func SetDefaults_KubeletConfiguration(obj *KubeletConfiguration) {
 		obj.HairpinMode = PromiscuousBridge
 	}
 	if obj.EvictionHard == nil {
-		temp := "memory.available<100Mi"
+		temp := "memory.available<300Mi,nodefs.available<10%,nodefs.inodesFree<5%"
```

Comment:
how was this reached?  i left kubecon thinking we did not have good defaults in mind?

@sjenning